source/Sashimi.Tests.Shared/Sashimi.Tests.Shared.csproj

```diff
@@ -13,7 +13,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Calamari.Terraform\Calamari.Terraform.csproj" />
```

Comment:
Shouldn't need the `Terraform` reference

Reply:
Fixed